code/modules/mapping/mapping_helpers.dm

```diff
@@ -245,6 +245,139 @@
 	else
 		airlock.autoname = TRUE
 
+//apc helpers
+/obj/effect/mapping_helpers/apc
+	desc = "You shouldn't see this. Report it please."
+	layer = DOOR_HELPER_LAYER
+	late = TRUE
+
+/obj/effect/mapping_helpers/apc/Initialize(mapload)
+	. = ..()
+	if(!mapload)
+		log_mapping("[src] spawned outside of mapload!")
+		return
```

Comment:
Do _not_ log this to mapping.log if this happens since it is likely an admin putting it there, and the mapping log should only be for direct mapping issues, not anything that happens in a live shift.

You may stack_trace it if you really want to, but I don't really think we're losing anything if we just make this a simple `return INITIALIZE_HINT_QDEL`.

Reply:
```suggestion
	if(!mapload)
		log_mapping("[src] spawned outside of mapload!")
		return INITIALIZE_HINT_QDEL
```

this should be done at the very least, to ensure we don't waste time doing LateInitialize stuff (unless you really want to).